orbitize/hipparcos.py

```diff
@@ -390,8 +393,19 @@ def log_prob(model_pars):
         axes[4].plot(xs, norm(0, myHipLogProb.delta0_err).pdf(xs), color='k')
         axes[4].set_xlabel('Dec Offset [mas]')
 
-
         plt.tight_layout()
         plt.savefig(saveplot, dpi=250)
 
-    return sampler.flatchain, myHipLogProb
\ No newline at end of file
+    return sampler.flatchain, myHipLogProb
+
+
+if __name__ == '__main__':
```

Comment:
Did you mean to include this snippet with `if __name__ == '_main__':`. Looks like a mini debugging test.

Reply:
Nope, I'll remove it. Thanks!